### PR TITLE
sm64ex: Fix server

### PR DESCRIPTION
**Pull request by N00byKing**
Holdover from #2583.

## What is this fixing or adding?
SM64Levels enum does not seem to be available post-generation?
This fixes it.

## How was this tested?
Running multiserver again

### CODE REVIEW

**Comment by N00byKing** [comment]
Note: Change already included in #2569.
Do not merge this if #2569 is also to be merged

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
